pkg/provider/resource_pgd.go

```diff
@@ -376,6 +381,11 @@ func PgdSchema(ctx context.Context) schema.Schema {
 							ElementType:   types.StringType,
 							PlanModifiers: []planmodifier.Set{setplanmodifier.UseStateForUnknown()},
 						},
+						"read_only_connections": schema.BoolAttribute{
+							Description: "Read-only connections.",
```

Comment:
Change to something like `Is read-only connections enabled.`

Reply:
Fixed in the latest commit.